tiles-generation/dockerAssets/generate_tiles.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+
+# Adapted from https://svn.openstreetmap.org/applications/rendering/mapnik/generate_tiles.py
+
+from math import pi,sin,log,exp,atan
+import os
+from queue import Queue
+import logging
+
+import threading
+
+try:
+    import mapnik2 as mapnik
+except:
+    import mapnik
+
+DEG_TO_RAD = pi/180
+RAD_TO_DEG = 180/pi
+
+def minmax (a,b,c):
+    a = max(a,b)
+    a = min(a,c)
+    return a
+
+class GoogleProjection:
```

Comment:
Why it is called GoogleProjection?

Reply:
It's a standard map projection to render map, and this script is from OSM.